BlazarData/src/test/java/com/hubspot/blazar/data/service/BranchServiceTest.java

```diff
@@ -33,18 +33,20 @@ public void testUpsertBasic() throws Exception {
     assertThat(retrieved.isPresent()).isTrue();
     assertThat(retrieved.get()).isEqualTo(inserted);
     assertThat(retrieved.get().getCreatedTimestamp()).isBetween(before, System.currentTimeMillis());
-    assertThat(retrieved.get().getUpdatedTimestamp()).isBetween(before, System.currentTimeMillis());
-    assertThat(retrieved.get().getUpdatedTimestamp()).isEqualTo(retrieved.get().getCreatedTimestamp());
+
+    assertThat(retrieved.get().getUpdatedTimestamp())
+        .isBetween(before, System.currentTimeMillis())
+        .isEqualTo(retrieved.get().getCreatedTimestamp());
   }
 
   @Test
-  public void testUpsertRepositoryRename() throws InterruptedException {
+  public void itUpdatesARenamedBranchWhenAlreadyExists() throws InterruptedException {
     long before = System.currentTimeMillis();
     GitInfo original = newGitInfo(123, "Overwatch", "master");
     original = branchService.upsert(original);
-    // The last updated column is `TIMESTAMP` and as such is 1s precision
+    // The last updated column is type `TIMESTAMP` and is 1s precision
+    // We have to wait so that the updated timestamp is actually greater
```

Comment:
Not worth to delay the test by sleeping just to check the updated timestamp. You can set `before` to System.currentTimeMillis() - 1000